client/src/locales/strings.json

```
@@ -27,6 +28,9 @@
     },
     "createAccountButton": {
       "text": "Crear Cuenta"
+    },
+    "logInButton": {
+      "text": "Iniciar sesión"
```

Nit:
```suggestion
      "text": "Iniciar Sesión"
```
Para ser consistentes con los demás...